airflow/models/dagrun.py

```diff
@@ -75,6 +80,8 @@ class DagRun(Base, LoggingMixin):
         backref=backref('dag_run', uselist=False),
     )
 
+    DEFAULT_DAGRUNS_TO_EXAMINE = airflow_conf.getint('scheduler', 'max_dagruns_per_query', fallback=20)
```

Comment:
numbers picked entirely arbitrarily and it seemed to permit will in tests.

The other number is hard coded still because I haven't addressed the to-do comment I already left to pull it out